repos/system_upgrade/el7toel8/actors/removedpammodules/actor.py

```diff
@@ -0,0 +1,48 @@
+from leapp.actors import Actor
+from leapp.models import Report, PamConfiguration
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.libraries.stdlib import api
+from leapp.libraries.common.reporting import report_with_remediation
+
+
+class RemovedPamModules(Actor):
```

Comment:
Please call this actor CheckRemovedPamModules

Reply:
And please also update the path to be called checkremovedpammodules

Reply:
Will do.